packages/eslint-plugin/src/configs/strict.ts

```diff
@@ -29,6 +29,7 @@ export = {
     '@typescript-eslint/no-unnecessary-condition': 'warn',
     '@typescript-eslint/no-unnecessary-type-arguments': 'warn',
     '@typescript-eslint/no-unsafe-declaration-merging': 'warn',
+    '@typescript-eslint/no-unsafe-enum-comparison': 'warn',
```

Comment:
Ah wait, per https://typescript-eslint.io/linting/configs, `strict` isn't considered stable. It's not in our "recommended" configs list. 😌 

Still filing #6890 just in case.